initialization.tex

```diff
@@ -336,9 +336,9 @@ \subsection{Perturbation State}
 \end{equation}
 
 \noindent Starting with the reference state fields 
-(\ref{init_pbar}, \ref{init_alphabar}, and \eqref{init_mubar}) and the perturbation (\ref{init_muprime}),
+(\ref{init_pbar}, \ref{init_alphabar}, and \ref{init_mubar}) and the perturbation (\ref{init_muprime}),
```

Comment:
@davegill 
Did you change this? If I did, I didn't mean to. Just want to make sure it's correct.

Reply:
I did not change it either. Let's see what the pdf looks like afterwards.